resources/en/view/Answer_Input.view.bxb

```diff
@@ -3,10 +3,14 @@ input-view {
     to-input: UpdateQuiz(action)
   }
   
+  // Mix and match the Macros to get your desired dialog. Macros include:
+  // score-statement: Says correct or incorrect
+  // quiz-show-correct: Tells the correct answer if the user answered incorrectly
+  // quiz-questions-left-fragment: How many questions are left
   message {
     if (exists(action.state.lastScore)) {
-      template ("#{macro('score-statement', action.state.lastScore)} #{value(action.state.currentQuestion.textToDisplay)}") {
-        speech ("#{macro('score-statement', action.state.lastScore)}... #{value(action.state.currentQuestion.textToSpeak)}")
+      template ("#{macro('score-statement', action.state.lastScore)} #{macro('quiz-show-correct', action.state.lastScore)} #{macro('quiz-questions-left-fragment', action.state.scoreBook.questionsLeftCount)} #{value(action.state.currentQuestion.textToDisplay)}") {
+        speech ("#{macro('score-statement', action.state.lastScore)} #{macro('quiz-show-correct', action.state.lastScore)} #{macro('quiz-questions-left-fragment', action.state.scoreBook.questionsLeftCount)}... #{value(action.state.currentQuestion.textToSpeak)}")
```

Comment:
Make sure TTS reads the speech channel okay when statements are concatenated. I found out that TTS is not pausing enough between the statements unless I add "..." between them.